dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -603,9 +657,8 @@ def test_profile(self):
             quantiles = profile.pop('quantiles')
             expected_quantiles = expected_profile.pop('quantiles')
             actual_quartiles = {0: quantiles[249], 1: quantiles[499], 2: quantiles[749]}
-
-            self.assertDictEqual(expected_profile, profile)
-            self.assertEqual(expected_profile['precision'], 1.0)
+            self.assertCountEqual(expected_profile, profile)
+            self.assertCountEqual(expected_profile['precision'], profile['precision'])
             self.assertCountEqual(expected_histogram['bin_counts'],
                                   histogram['bin_counts'])
             self.assertCountEqual(np.round(expected_histogram['bin_edges'], 12),
```

Comment:
these technically should be:
```python
self.assertDictEqual(expected_profile['precision'], profile['precision'])
            self.assertTrue(np.all(expected_histogram['bin_counts'] ==
                                  histogram['bin_counts']))
            self.assertTrue(np.all(np.round(expected_histogram['bin_edges'], 12) ==
                                  np.round(histogram['bin_edges'], 12)))

            self.assertDictEqual(actual_quartiles, expected_quantiles)
```

but i can fix these in a subsequent PR

Reply:
disregard this, countequal actually works and prevents disagreements between int/float which is very interesting.